dirsrvtests/tests/suites/plugins/dna_interval_test.py

```diff
@@ -15,12 +15,44 @@
 from lib389.idm.organizationalunit import OrganizationalUnits
 from lib389.idm.user import UserAccounts
 from lib389.topologies import topology_st
+from lib389.utils import *
 
 pytestmark = pytest.mark.tier1
 
 log = logging.getLogger(__name__)
 
-def test_dna_interval(topology_st):
+
+@pytest.fixture(scope="function")
+def dna_plugin(topology_st, request):
+    inst = topology_st.standalone
+    plugin = DNAPlugin(inst)
+    ous = OrganizationalUnits(inst, DEFAULT_SUFFIX)
+    ou_people = ous.get("People")
+
+    log.info("Add dna plugin config entry...")
+    configs = DNAPluginConfigs(inst, plugin.dn)
+    dna_config = configs.create(properties={'cn': 'dna config',
+                                            'dnaType': 'uidNumber',
+                                            'dnaMaxValue': '1000',
+                                            'dnaMagicRegen': '-1',
+                                            'dnaFilter': '(objectclass=top)',
+                                            'dnaScope': ou_people.dn,
+                                            'dnaNextValue': '10',
+                                            'dnaInterval': '10'})
+
+    log.info("Enable the DNA plugin and restart...")
+    plugin.enable()
+    inst.restart()
+
+    def fin():
+        dna_config.delete()
```

Comment:
Need to find a better way of removing the plugin's config as it is making the tests flaky. Sometimes it is making the server go under a shutdown state. A solution which I can think of: we should check if the server is up and if yes then shut it down remove the entries and restart it.